Foundation/FileManager.swift

```diff
@@ -618,7 +793,28 @@ open class FileManager : NSObject {
         })
         return contents
     }
-    
+
+#if os(Windows)
+    private func joinPath(prefix: String, suffix: String) -> String {
+      let pszPath: UnsafeMutablePointer<PWSTR?> =
+          UnsafeMutablePointer<PWSTR?>(bitPattern: 0)!
+
+      prefix.withCString(encodedAs: UTF16.self) { prefix in
+        suffix.withCString(encodedAs: UTF16.self) { suffix in
+          guard PathAllocCombine(prefix, suffix,
+                                 ULONG(PATHCCH_ALLOW_LONG_PATHS.rawValue),
+                                 pszPath) == S_OK else {
+            return String()
+          }
+        }
+      }
+
+      let path: String = String(decodingCString: pszPath.pointee!, as: UTF16.self)
```

Comment:
```suggestion
      let path: String = String(decodingCString: pszPath!.pointee!, as: UTF16.self)
```

Reply:
Too many layers of unwrapping.  I think it should be `pszPath!.pointee`.